ec-europa/eubfr-data-lake
# refactor(etl): add types to all transform functions - EUBFR-108

Notes
- added `flow-watch` as it's quite hard to be productive otherwise
- the `Location` shape has been changed drastically, after discussions
- `Project` interface has all fields encountered in all ETLs

Tests:
- ensure elasticsearch can persist data for dates and location specifically
- check for any missing fields that I might have missed
